_data/meltano/utilities/dbt-bigquery/dbt-labs.yml

```
@@ -0,0 +1,134 @@
+name: dbt-bigquery
+label: dbt BigQuery (EDK preview)
+executable: dbt_invoker
+namespace: dbt_bigquery
+docs: https://docs.meltano.com/guide/transformation
+repo: https://github.com/dbt-labs/dbt-bigquery
+ext_repo: https://github.com/meltano/dbt-ext
+pip_url: dbt-core~=1.1.0 dbt-bigquery~=1.1.0 git+https://github.com/meltano/dbt-ext.git@main
```

```suggestion
pip_url: dbt-core~=1.3.0 dbt-bigquery~=1.3.0 git+https://github.com/meltano/dbt-ext.git@main
```
@pandemicsyn any reasons we couldn't use the latest for all of these? Looking at https://github.com/dbt-labs/dbt-core/releases?page=2 it doesn't seem there are relevant breaking changes in 1.2 or 1.3.

---

@tayloramurphy Should be fine, some of them where already 1.2 anyway. I'll send a long an update to bump them all to 1.3